spec/lib/auth0/api/v2/client_grants_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+describe Auth0::Api::V2::ClientGrants do
+  before :all do
+    dummy_instance = DummyClass.new
+    dummy_instance.extend(Auth0::Api::V2::ClientGrants)
+    @instance = dummy_instance
+  end
+  context '.client_grants' do
+    it { expect(@instance).to respond_to(:client_grants) }
+    it { expect(@instance).to respond_to(:get_all_client_grants) }
+    it 'is expected to send get request to /api/v2/client_grants/' do
+      expect(@instance).to receive(:get).with('/api/v2/client-grants')
+      expect { @instance.client_grants }.not_to raise_error
+    end
+  end
+
+  context '.create_client_grant' do
+    it { expect(@instance).to respond_to(:create_client_grant) }
+    it 'is expected to send post to /api/v2/client-grants' do
+      expect(@instance).to receive(:post).with('/api/v2/client-grants', opt: 'test body')
+      expect { @instance.create_client_grant(opt: 'test body') }.not_to raise_error
```

Comment:
@ignaciojonas The method would try to parse the parameter as a hash, with key-value pairs. Since a plain string would fail, I used the 'opt' key. I picked the 'opt' name from the clients spec btw.